decompiler/pipeline/controlflowanalysis/readability_based_refinement.py

```diff
@@ -428,7 +428,7 @@ def rename(self):
             if not isinstance(loop_node.declaration, Assignment):
                 continue
 
-            old_variable: Variable = loop_node.declaration.destination
+            old_variable: Variable = self._get_variable_from_assignment(loop_node.declaration.destination)
```

Comment:
Please add a test to readability refinement.

Reply:
Done, try to rename a for loop where the decl. is a ListOp